.github/workflows/sanity-upgrade-test.yml

```diff
@@ -0,0 +1,310 @@
+---
+# SPDX-FileCopyrightText: (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+name: Upgrade Test
+
+on:
+  pull_request_review_comment:
+    types: [created]  # Run on PR review comment "upgrade-test"
```

Comment:
Are you sure about using this? I found many issues in using this trigger. It basically gets triggered on every single new comment in a PR. I don't think we want that.